src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/HelperTest.java

```diff
@@ -140,6 +141,48 @@ public void hearingMidEventValidationDayError() {
 
     }
 
+    @Test
+    public void validateHearingDatesNotInFutureTest() {
```

Comment:
Would you tests be more readable if you had two methods, one to test for dates in the future and another to test for dates in the past?